VagrantProvision.sh

```diff
@@ -88,6 +88,10 @@ sudo apt-get -y autoremove
 
 echo "### Configuring environment..."
 
+# Configure https alternative mirror for maven isntall, this can likely be removed once
+# we are using maven 3.2.3 or higher
+sudo $HOOT_HOME/scripts/SetMavenHttps.sh
```

Comment:
The script is a Perl script and should actually be invoked by something like this:
`sudo /usr/bin/perl $HOOT_HOME/scripts/SetMavenHttps.pl`
...and then rename the file from `SetMavenHttps.sh` which is the extension for shell scripts to `SetMavenHttps.pl` which is the correct extension for Perl scripts, which it is.

The actual Perl script looks good and it executes correctly the way you have it but it is very unorthodox.